ui/tests/helpers/pki/policy-generator.js

```diff
@@ -0,0 +1,52 @@
+import { singularize } from 'ember-inflector';
```

Comment:
I love this policy generator pattern! I feel like it might be easier to extend if we put this file under `tests/helpers/policy-generator/pki.js` instead, so then all the policies are in one folder

Reply:
oh great idea! will do!